cpp/src/ice2slice/Gen.cpp

```diff
@@ -32,29 +32,23 @@ namespace
         return cont->getMetadataArgs("cs:identifier").value_or(cont->name());
     }
 
```

Comment:
This is only tangentially related to my PR. I merged `getCsMappedScoped` into `getCsMappedScope`, since that was the only place it was being called from, and the function was a one-liner.

Also, we _never_ want a leading `.` in this context, so I made the same change as I did in `libSlice`, but assuming that `leading` is always `false`.